cluster/manifests/skipper/daemonset.yaml

```diff
@@ -25,6 +25,8 @@ spec:
           [{"container": "skipper-ingress", "parser": "skipper-access-log"}]
     spec:
       priorityClassName: system-node-critical
+      - key: dedicated
```

Comment:
Shouldn't the key name come from the config item?

Reply:
I didn't know this was intended. The key is arbitrary and lives solely in the cluster registry.

The current approach:
* config items allow setting arbitrary taints on node pools.
* `skipper` tolerates any dedicated nodes in the cluster which means nodes dedicated to anything (gpus, local disks, particular teams, used for performance tests, ...) will still run `skipper` because it's a cluster component (the meaning of dedicated is derived from https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/ "Example Use Cases")

If we wanted to tolerate it dynamically, it would essentially mean: "tolerate any taint that is defined in the cluster registry via a config item on a worker node pool". Is this what we want?

Reply:
> Is this what we want?

That's what I imagined. Either skipper is special in the sense that it should not be scheduled on "special" nodes or it's like all the other daemonsets IMO.

WDYT?

Reply:
I'd suggest not running it on dedicated nodes (and switch a normal deployment in the future, which would solve the issue completely).

Reply:
@mikkeloscar DaemonSets support node selectors, so I wonder if we could achieve the same by running `skipper` DS with the `kubernetes.io/role=worker` node selector.